content/fr/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,231 @@
+| `DiskPressure`       | `True` s'il existe une pression sur la taille du disque, c'est-à-dire si la capacité du disque est faible; autrement `False`                                                                                                                                                  |
+| `NetworkUnavailable` | `True` si le réseau pour le noeud n'est pas correctement configuré, sinon `False`                                                                                                                                                                      |
+
+La condition de noeud est représentée sous la forme d'un objet JSON.
+Par exemple, la réponse suivante décrit un nœud sain.
+
+```json
+"conditions": [
+  {
+    "type": "Ready",
+    "status": "True"
+  }
+]
+```
+
+Si le statut de l'état Ready reste `Unknown` ou `False` plus longtemps que `pod-eviction-timeout`, un argument est passé au [kube-controller-manager](/docs/admin/kube-controller-manager/) et les pods sur le nœud sont programmés pour être supprimés par le contrôleur du nœud.
+Le délai d’expulsion par défaut est de **cinq minutes**..
+Dans certains cas, lorsque le nœud est inaccessible, l'apiserver est incapable de communiquer avec le kubelet sur le nœud.
+La décision de supprimer les pods ne peut pas être communiquée au kublet tant que la communication avec l'apiserver n'est pas rétablie.
+Entre-temps, les pods dont la suppression est planifiée peuvent continuer à s'exécuter sur le nœud inaccessible.
+
+Dans les versions de Kubernetes antérieures à 1.5, le contrôleur de noeud [forcait la suppression](/docs/concepts/workloads/pods/pod/#force-deletion-of-pods) de ces pods inaccessibles de l'apiserver.
+Toutefois, dans la version 1.5 et ultérieure, le contrôleur de noeud ne force pas la suppression des pods tant qu'il n'est pas confirmé qu'ils ont cessé de fonctionner dans le cluster.
+Vous pouvez voir que les pods en cours d'exécution sur un nœud inaccessible sont dans l'état `Terminating` ou` Unknown`.
+Dans les cas où Kubernetes ne peut pas déduire de l'infrastructure sous-jacente si un nœud a définitivement quitté un cluster, l'administrateur du cluster peut avoir besoin de supprimer l'objet nœud à la main.
+La suppression de l'objet nœud de Kubernetes entraîne la suppression de tous les objets Pod exécutés sur le nœud de l'apiserver et libère leurs noms.
+
+Dans la version 1.12, la fonctionnalité `TaintNodesByCondition` est promue en version bêta, ce qui permet au contrôleur de cycle de vie du noeud de créer automatiquement des [marquages (taints en anglais)](/docs/concepts/configuration/taint-and-toleration/) qui représentent des conditions.
+De même, l'ordonnanceur ignore les conditions lors de la prise en compte d'un nœud; au lieu de cela, il regarde les taints du nœud et les tolérances d'un pod.
```

Comment:
jfyi - not sure if we keep "taints"
there is a few more below

Reply:
Je serais d'avis d'aller avec marquer/marquage/marque parce que taint en anglais a une connotation plus forte qu'en français qui est simplement lié à une couleur. cf discussion sur slack